app/dashboard/templates/onepager/base.html

```diff
@@ -129,8 +131,9 @@
     {% include 'shared/messages.html' %}
 
     <script>
+      var host = ("{{ ipfs_config.host }}" == "ipfs" ? "localhost" : "{{ ipfs_config.host }}");
```

Comment:
Can we adjust this to `ipfsHost` or something like that to avoid potential collisions?